docs/api/api_alternative.md

```diff
@@ -0,0 +1,289 @@
+| [InterruptIn](../apis/interruptin.html) | &#10004; | &#10004;  |
+
+### USB drivers
+
+The Mbed OS classes providing USB peripheral functionality, also known as USB components, inherit from USBDevice and provide specific USB peripherial functionality.
+
+| API | Full profile | Bare metal profile |
+| - | - | - |
+| [USBSerial](../apis/usb-apis.html) | &#10004; | &#10004;  |
+| [USBAudio](../apis/usbaudio.html) | &#10004; | &#10004;  |
+| [USBHID](../apis/ubshid.html) | &#10004; | &#10004;  |
+| [USBCDC](../apis/usbcdc.html) | &#10004; | &#10004;  |
+| [USBCDC_ECM](../apis/usbcdc-ecm.html) | &#10004; | &#10004;  |
+| [USBMSD](../apis/usbmsd.html) | &#10004; | &#10004; |
+| [USBMIDI](../apis/usbmidi.html) | &#10004; | &#10004;  |
+| [USBMouse](../apis/usbmouse.html) | &#10004; | &#10004;  |
+| [USBKeyboard](../apis/usbkeyboard.html) | &#10004; | &#10004;  |
+| [USBMouseKeyboard](../apis/usbmousekeyboard.html) | &#10004; | &#10004;  |
+
+### Other drivers
+
+The drivers are digital interfaces that allow your board to interface with a computer or external devices.
+
+| API | Full profile | Bare metal profile |
+| - | - | - |
+| [Timeout](../apis/timeout.html) | &#10004; | &#10004; |
+| [Watchdog](../apis/watchdog.html) | &#10004; | &#10004; |
+| [ResetReason](../apis/resetreason.html) | &#10004; | &#10004;  |
+| [Flash IAP](../apis/flash-iap.html) | &#10004; | &#10004;  |
+| [BufferedSerial](../apis/bufferedserial.html) | &#10004; | &#10004;  |
+| [UnbufferedSerial](../apis/unbufferedserial.html) | &#10004; | &#10004;  |
+| [SPI](../apis/spi.html) | &#10004; | &#10004;  |
+| [SPISlave](../apis/spislave.html) | &#10004; | &#10004;  |
+| [QuadSPI (QSPI)](../apis/quadspi-qspi.html) | &#10004; | &#10004; |
+| [I2C](../apis/i2c.html) | &#10004; | &#10004;  |
+| [I2CSlave](../apis/i2cslave.html) | &#10004; | &#10004;  |
+| [CAN](../apis/can.html)| &#10004; | &#10004;  |
+| [MbedCRC](../apis/mbedcrc.html) | &#10004; | &#10004;  |
+
+
+## Platform
+
+### Power
+
+<!--needs intro-->
+
+| API | Full profile | Bare metal profile |
+| - | - | - |
+| [Power management (sleep)](../apis/power-apis.html) | &#10004; | &#10004; |
+| [DeepSleepLock](../apis/deepsleeplock.html) | &#10004; | &#10004;  |
+| [Idle loop](../apis/idle-loop.html) | &#10004; | &#10004;  |
+| [LowPowerTicker](../apis/lowpowerticker.html) | &#10004; | &#10004;  |
+| [LowPowerTimeout](../apis/lowpowertimeout.html) | &#10004; | &#10004;  |
+| [LowPowerTimer](../apis/lowpowertimer.html) | &#10004; | &#10004;  |
+
+### Memory
+
+<!--needs intro-->
+
+| API | Full profile | Bare metal profile |
+| - | - | - |
+| [mbed_stats (Mbed statistics)](../apis/mbed-statistics.html) | &#10004; | &#10004;  |
+| [mpug_mgmt (MPU management)](../apis/mpu-management.html) | &#10004; | &#10004;  |
+| [mbed_mem_trace (Memory tracing)](../apis/memory-tracing.html) | &#10004; | &#10004;  |
+| [MemoryPool](../apis/memorypool.html) | &#10004; |   |
+
+### Time
+
+| API | Full profile | Bare metal profile |
```

Comment:
Could you give the API column a minimum width?

Reply:
I'm afraid not - we don't control the way the tables are rendered